internal/controller/compute/statefulserverset/datavolume_controller.go

```diff
@@ -0,0 +1,165 @@
+package statefulserverset
```

Comment:
Could you please create some unit tests for this controller?

Reply:
I will add separate test in story: https://hosting-jira.1and1.org/browse/VDC-16027

Reply:
Thanks!